docs/querying/api.md

```diff
@@ -24,6 +24,10 @@ and one of the following HTTP response codes:
 Other non-`2xx` codes may be returned for errors occurring before the API
 endpoint is reached.
 
+An array of warnings will be returned if there is an error while querying
+remote storage sources. All data from those sources are excluded, but data
```

Comment:
Let's not tie this to remote storage, this part of the docs is the general response format and we may have other uses for this in future (e.g. ignoring corrupted local chunks).